packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakeEthForm/Inputs.tsx

```diff
@@ -71,42 +81,108 @@ export const Inputs = () => {
     const shouldShowAmountForWithdrawalWarning =
         isLessAmountForWithdrawalWarningShown || isAmountForWithdrawalWarningShown;
 
+    const isFractionButtonDisabled = (divisor: number) => {
+        if (!account.formattedBalance || !network.decimals) return false;
+
+        return new BigNumber(account.formattedBalance)
+            .dividedBy(divisor)
+            .decimalPlaces(network.decimals)
+            .lte(MIN_ETH_AMOUNT_FOR_STAKING);
+    };
+
+    const tooltip = (
+        <Translation
+            id="TR_STAKE_MIN_AMOUNT_TOOLTIP"
+            values={{
+                amount: MIN_ETH_AMOUNT_FOR_STAKING.toString(),
+                symbol: account.symbol.toUpperCase(),
+            }}
+        />
+    );
+
+    const isBalanceBelowMinStake = new BigNumber(account.formattedBalance || '0').lt(
+        MIN_ETH_AMOUNT_FOR_STAKING,
+    );
+
     return (
-        <Column gap={spacings.sm} alignItems="center">
-            <NumberInput
-                name={CRYPTO_INPUT}
-                labelLeft={
-                    <FormFractionButtons
-                        setRatioAmount={setRatioAmount}
-                        setMax={setMax}
-                        symbol={account.symbol}
-                        totalAmount={account.formattedBalance}
-                        decimals={network.decimals}
+        <Column gap={spacings.sm}>
+            <InputWithOptions
+                input={
```

Comment:
Do we need this? It seems it's always a NumberInput, since it has the fraction buttons.